apps/api/src/app/environments-v1/usecases/output-renderers/novu-branding-html.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * This is the HTML for the Novu branding image.
+ * Should be in par with the actual React component we show in the dashboard preview:
+ * @see apps/dashboard/src/components/workflow-editor/steps/email/novu-branding.tsx
+ */
+export const NOVU_BRANDING_HTML = `
+<table align="center" width="100%" border="0" cellpadding="0" cellspacing="0" role="presentation" style="max-width:600px;min-width:300px;width:100%;margin:24px auto 8px auto;">
+  <tbody>
+    <tr style="width:100%">
+      <td align="center">
+        <a href="https://novu.co?utm_source=email&utm_medium=branding&utm_campaign=powered_by" target="_blank" rel="noopener noreferrer" style="text-decoration:none;">
```

Comment:
@scopsy Let me know if these are the right UTM params to track

Reply:
Please use dub.co/powered?utm_source=email

Are you sure? Inbox branding should also use dub.co

Reply:
dub.co/powered?utm_source=email

Reply:
Its seems like we use `go.novu.co` in the inbox, will recheck

```tsx
      <a
              href={`https://go.novu.co/powered?ref=${getCurrentDomain()}`}
              target="_blank"
              class="nt-z-10 nt-flex nt-items-center nt-gap-1 nt-justify-center"
            >
              <span class="nt-text-xs">Inbox by</span>
              <Novu class="nt-size-2.5" />
              <span class="nt-text-xs">Novu</span>
            </a>
     ```